cmd/cored/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # cored changelog
 
+## 1.1.0 (February 24, 2017)
+
+This release is a minor version update, and contains **new features** and **deprecations**. cored 1.1.0 is backward-compatible with 1.0.x SDKs, but we strongly recommend upgrading to 1.1.x SDKs as soon as possible. See SDK-level changes here:
+
+- [Java SDK changelog](https://github.com/chain/chain/blob/main/sdk/java/CHANGELOG.md)
+- [Node SDK changelog](https://github.com/chain/chain/blob/main/sdk/node/CHANGELOG.md)
+- [Ruby SDK changelog](https://github.com/chain/chain/blob/main/sdk/ruby/CHANGELOG.md)
+
+Other changes:
```

Comment:
"Notable changes" instead? Doesn't feel like there's any changes above this to make it "other" except for the version number.